application/views/template.php

```diff
@@ -45,12 +45,27 @@
 								echo $content;
 							}
 						} else {
-							echo 'Page does not have any content';
+							$services_down;
```

Comment:
In php we usually don't declare empty variables. But it's good that you set it outside the if's. 

If I would have done this I would have given it a value, because if something goes wrong, that we don't know about, there could be an error message saying `The OP5 Monitor , please contact your administrator.`

One way would be to maybe declare this variable as `$services_down = 'services LMD and Naemon are not running` and remove the if/else, and maybe extend the if's inside the else